lib/p2p/Peer.ts

```diff
@@ -76,6 +79,7 @@ class Peer extends EventEmitter {
   }
 
   public get info(): PeerInfo {
+    this.emitError;
```

Comment:
What is this line doing?

Reply:
Totally inadvertent, thanks for catching.